internal/encryption/method/unencrypted/method.go

```diff
@@ -0,0 +1,38 @@
+package unencrypted
+
+import (
+	"github.com/opentofu/opentofu/internal/encryption/method"
+)
+
+func New() method.Descriptor {
+	return &descriptor{}
+}
+
+type descriptor struct{}
+
+func (f *descriptor) ID() method.ID {
+	return "unencrypted"
+}
+func (f *descriptor) ConfigStruct() method.Config {
+	return new(config)
+}
+
+type config struct{}
+
+func (c *config) Build() (method.Method, error) {
+	return new(unenc), nil
+}
+
+type unenc struct{}
+
+func (a *unenc) Encrypt(data []byte) ([]byte, error) {
+	panic("Placeholder for type check!  Should never be called!")
+}
+func (a *unenc) Decrypt(data []byte) ([]byte, error) {
+	panic("Placeholder for type check!  Should never be called!")
```

Comment:
I'm missing something. Why are we using unencrypted.Is everywhere instead of just providing noop methods here that return what they receive without changes?

Reply:
Why does this need to be a special case other than some select validations.

Reply:
To setup decryption methods, you need the embedded metadata.  With an unencrypted payload, you don't have any metadata and can't setup the methods.

It's sort of a chicken and egg problem that can't really be handled outside of base.go

Reply:
Ah, fair enough.